_posts/2022-05-12-data-pipelines-in-Databricks.markdown

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title: "Data pipelines in Databricks"
+date: 2022-05-12
+tags: Databricks data testing
+author: Natalie Laing
+image: /img/code.jpg
+---
+
+# Data pipelines in Databricks
+
+## Introduction
+
+![delta lake](/img/blog/databricks-data-pipeline/delta-lake.png)
+<br />
+
+### Delta Lake
+
+What is a delta lake?
+
+Delta lake enables developers to create Lakehouse architectures on top of data lakes.
+
+Delta lake is open source.
+Delta lakehouse combines elements from a data warehouse and a data lake.
+
+![data lakehouse](/img/blog/databricks-data-pipeline/data-lakehouse.png)
+
+### Databricks
+
+What is Databricks?
+
+Databricks provides a solution for ingestion, processing and scheduling allowing developers to run ETL workloads directly on a data lake.
+
+What is a data pipeline in Databricks?
+
+A data pipeline is only concerned with the transformation of Databricks job code, this usually consists of moving data from bronze to gold classification.
+
+- Bronze
+  - Ingest raw data
+- Silver
+  - Deduplicate data
+  - Enforce schemas
+- Gold
+  - Perform business-specific transforms to data
+  - Aggregate data
+
+<br/>
+
+### Multi task jobs v delta live jobs
```

Comment:
Some introduction here also would be useful: 
e.g. 
In Databricks you could use two different methods for creating data pipelines:
1. Multi-task jobs
2. Delta live tables